src/components/ClearInputsButton/ClearInputsButton.css

```diff
@@ -0,0 +1,5 @@
+@import "../../index.css";
+
+.ClearInputsButton {
+  padding: 0px 0 0;
+}
```

Comment:
Try not to overwrite built-in button styles.

Reply:
Removed